examples/src/main/scala/org/apache/spark/examples/mllib/IsotonicRegressionExample.scala

```diff
@@ -55,7 +55,7 @@ object IsotonicRegressionExample {
     }
 
     // Calculate mean squared error between predicted and real labels.
-    val meanSquaredError = predictionAndLabel.map { case (p, l) => math.pow((p - l), 2) }.mean()
+    val meanSquaredError = predictionAndLabel.map { case (p, la) => math.pow((p - la), 2) }.mean()
```

Comment:
You could write out a little more, like pred and label, in cases like this. If we're going to expand them might as well do more.